File: 3872-find-most-frequent-vowel-and-consonant/3872-find-most-frequent-vowel-and-consonant.cpp
class Solution {
public:
    int maxFreqSum(string s) {
        int maxVowel = 0;
        int maxConst = 0;
        int alpha[26] = {0};
        for(int i = 0; i < s.size(); i++)
        {
            alpha[s[i] - 'a']++;
            if(s[i] == 'a' || s[i] == 'e' || s[i] == 'i' || s[i] == 'o' || s[i] == 'u')
            {
                maxVowel = max(maxVowel, alpha[s[i] - 'a']);
            }
            else
            {
                maxConst = max(maxConst, alpha[s[i] - 'a']);
            }
        }
        return maxVowel + maxConst;
    }
};

File: 3872-find-most-frequent-vowel-and-consonant/3872-find-most-frequent-vowel-and-consonant.py
class Solution:
    def maxFreqSum(self, s: str) -> int:
        maxVowel = 0
        maxConst = 0
        alpha = [0 for i in range(26)]
        for ch in s:
            alpha[ord(ch) - ord('a')] += 1
            if ch in {'a','e','i','o','u'}:
                maxVowel = max(maxVowel, alpha[ord(ch) - ord('a')])
            else:
                maxConst = max(maxConst, alpha[ord(ch) - ord('a')])
        return maxVowel + maxConst